js/test/test.js

```diff
@@ -411,7 +411,6 @@ let testMyTrades = async (exchange, symbol) => {
         // log (asTable (trades))
 
     } else {
-
```

Comment:
The test.js does not get transpiled anyway, so it is much less restrictive on coding style ;)

Reply:
That is just to retrigger the testsuite as you have just merged all the fixes for failed `fetchTrades()`. They should all pass now.

Reply:
Actually, I'm quite surprised these deep tests discovered only four issues out of fourscore exchanges. You've done pretty much accurate implementations without any tests )

Reply:
Without reports coming from community and without your help it would be much harder, just look at how many bugs were fixed in the issues, it was a long way, thx for your effort ) There's still a lot of room for work, enhancements, and, possibly, *a few bugs* as well.